### DISCUSSION TITLE
[Risolto] Far comunicare MQTT e ZHA

### DISCUSSION THREAD

**Discussion by TimVoda** [indomus/forum#2826] [GENERALE - Discussioni generiche]
Ciao a tutti, 
 cercavo un modo (o la conferma che non c' è) di far sì che il broker Mosquitto produca degli effetti sulle entità sotto ZHA (es. arriva a MQTT un messaggio che fa accendere una luce sotto ZHA)
Zigbee2MQTT non è un' opzione perchè mi toccherebbe eliminare ZHA e rifare tutto il sistema, se non ho capito male.
Esiste qualcosa del genere a vostra conoscenza?
Grazie

**Comment by snakuzzo** [comment]
Certo che puoi farlo.
Fai un'automazione su HA con trigger mqtt e action che va a pilotare l'entità o il device che hai su ZHA

**Comment by TimVoda** [comment]
Grazie.
Mi metto a studiare..
